arcgis-ios-sdk-samples/Edit data/Add features with contingent values/AddFeaturesContingentValuesViewController.swift

```diff
@@ -0,0 +1,200 @@
+                    self.queryFeatures()
+                }
+            }
+        case .failure(let error):
+            presentAlert(error: error)
+        }
+    }
+    
+    /// Creates a map with a vector tiled layer basemap.
+    func makeMap() -> AGSMap {
+        // Get the vector tiled layer by name.
+        let fillmoreVectorTiledLayer = AGSArcGISVectorTiledLayer(name: "FillmoreTopographicMap")
+        // Use the vector tiled layer as a basemap.
+        let basemap = AGSBasemap(baseLayer: fillmoreVectorTiledLayer)
+        let map = AGSMap(basemap: basemap)
+        return map
+    }
+    
+    /// Create buffer graphics for the features.
+    func queryFeatures() {
+        // Create the query parameters.
+        let queryParameters = AGSQueryParameters()
+        // Set the where clause to filter for buffer sizes greater than 0.
+        queryParameters.whereClause = "BufferSize > 0"
+        // Create an array of graphics to add to the graphics overlay.
+        var graphics = [AGSGraphic]()
+        // Query the features with the query parameters.
+        featureTable?.queryFeatures(with: queryParameters) { [weak self ] result, error in
+            guard let self = self else { return }
+            if let error = error {
+                // Present an alert if there is an error while querying the features.
+                self.presentAlert(error: error)
+            } else if let result = result {
+                // Get the array of features from the query result.
+                let features = result.featureEnumerator().allObjects
+                // For each feature, add the buffer symbol according to its buffer size.
+                features.forEach { feature in
+                    let graphic = self.createGraphics(for: feature)
+                    graphics.append(graphic)
+                }
+                // Add the graphics to the graphics overlay.
+                self.graphicsOverlay.graphics.addObjects(from: graphics)
+                // Add the graphics overlay to the map view.
+                self.mapView.graphicsOverlays.add(self.graphicsOverlay)
+            }
+        }
+    }
+    
+    /// Add a single feature to the map.
+    func addFeature(at mapPoint: AGSPoint) {
+        // Create a symbol to represent a bird's nest.
+        let symbol = AGSSimpleMarkerSymbol(style: .circle, color: .black, size: 11)
+        // Add the graphic to the graphics overlay.
+        graphicsOverlay.graphics.add(AGSGraphic(geometry: mapPoint, symbol: symbol))
+        // Show the attributes table view.
+        performSegue(withIdentifier: "AddFeature", sender: self)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Set the map view's touch delegate.
+        mapView.touchDelegate = self
+        
+        // Add the source code button item to the right of navigation bar.
+        (navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = [
```

Comment:
```suggestion
        (navigationItem.rightBarButtonItem as? SourceCodeBarButtonItem)?.filenames = [
```
It's highly unlikely that we'll change the UI structure of this sample viewer, but using ❓ is safer.